tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -558,8 +559,8 @@ func (a *AwsProvider) getPublicSubnetId(infraID string, vpc *ec2.Vpc) (string, e
 	return "", fmt.Errorf("could not find a public subnet in VPC: %v", *vpc.VpcId)
 }
 
-// GetInstance gets instance ec2 instance object from the given instanceID. We're making this method public
-// to use it in tests as of now.
+// GetInstance gets instance ec2 instance object from the given instanceID.
+// The function is exposed for e2e testing purpose.
```

Comment:
Why are you fixing comments in this PR? Please revert.

Reply:
Don't we want to call out why we are exposing functions in the comments? This is a function, although pushed by @ravisantoshgudimetla, still is exposed for testing purposes.